.github/workflows/test.yml

```diff
@@ -20,9 +20,9 @@ jobs:
           - target: aarch64-unknown-linux-gnu
             os: [self-hosted, linux, arm64]
           - target: x86_64-apple-darwin
-            os: macos-latest
+            os: macos-14
```

Comment:
There is no benefit to this change... Using `macos-latest` is simpler and requires less upkeep.

Reply:
It has been changed.